src/pages/project/project_wizard/ProjectForm.tsx

```diff
@@ -86,33 +87,68 @@ export const ProjectForm = ({ project, suggestedProjectManager }: ProjectFormPro
     <>
       <SkipLink href="#form">{t('common.skip_to_schema')}</SkipLink>
       <Formik initialValues={project} validationSchema={basicProjectValidationSchema} onSubmit={submitProjectForm}>
-        <Form noValidate>
-          <PageHeader>
-            <ProjectIconHeader projectStatus={projectWithId ? projectWithId.status : undefined} />
-            <TruncatableTypography variant="h1">{project.title}</TruncatableTypography>
-          </PageHeader>
-          <ProjectFormStepper
-            tabNumber={tabNumber}
-            setTabNumber={setTabNumber}
-            maxVisitedTab={maxVisitedTab}
-            setMaxVisitedTab={setMaxVisitedTab}
-          />
-          <RequiredDescription />
-          <Box sx={{ bgcolor: 'secondary.dark', padding: '0' }}>
-            <Box id="form" sx={{ bgcolor: 'secondary.main', mb: '0.5rem', padding: '1.5rem 1.25rem' }}>
-              {tabNumber === ProjectTabs.Description && <ProjectDescriptionForm thisIsRekProject={thisIsRekProject} />}
-              {tabNumber === ProjectTabs.Details && <ProjectDetailsForm thisIsRekProject={thisIsRekProject} />}
-              {tabNumber === ProjectTabs.Contributors && (
-                <ProjectContributorsForm
-                  maxVisitedTab={maxVisitedTab}
-                  suggestedProjectManager={suggestedProjectManager}
+        {({ setTouched }: FormikProps<SaveCristinProject>) => {
+          const onTabChange = (tab: ProjectTabs) => {
+            let maxTab = maxVisitedTab;
+            if (tab > maxVisitedTab) {
+              maxTab = tab;
+              setMaxVisitedTab(maxTab);
+            } else if (tab <= maxVisitedTab) {
+              // We have gone to a previous tab - we want maxTab to be validated as well
+              maxTab = maxVisitedTab + 1;
+            }
+            const touchedFields = getTouchedFields(maxTab);
+            setTouched(touchedFields);
+            setTabNumber(tab);
+          };
+
+          const onClickNext = () => {
+            const newTabNumber = tabNumber + 1;
+            setTabNumber(newTabNumber);
+            onTabChange(newTabNumber);
+          };
+
+          const onClickPrevious = () => {
+            const newTabNumber = tabNumber - 1;
+            setTabNumber(newTabNumber);
+            onTabChange(newTabNumber);
+          };
```

Comment:
`onClickNext` og `onClickPrevious` ser ut til å gjøre samme bare at de har en ulik step, så kanskje de kunne vært slått sammen til `onNavigate = (step: number)` elns?

Bare om du også syns det er en forenkling, for det er eksplisitt og fint as is også!